src/components/Textarea/styles.ts

```diff
@@ -0,0 +1,76 @@
+import styled, { css, DefaultTheme } from 'styled-components';
+
+import ResizeTextarea from 'react-textarea-autosize';
+
+import * as LabelBase from '@radix-ui/react-label';
+
+import { TextareaProps } from '.';
+import { transparentize } from 'polished';
+
+export const Wrapper = styled.div`
+  color: white;
+`;
+
+export const Label = styled(LabelBase.Root)`
+  ${({ theme }) => css`
+    display: block;
+    margin-bottom: 5px;
+    text-transform: uppercase;
+    font-weight: ${theme.fontWeights.medium};
+  `}
+`;
+
+const textAreaInputVariants = (theme: DefaultTheme) => ({
+  disabled: css`
+    &:disabled {
+      background: ${transparentize('0.5', theme.colors.grey[100])};
```

Comment:
This color should be grey[500], and the placeholder color needs to be handled as well, it should be grey[300]